fundingmanager.go

```diff
@@ -1100,11 +1127,24 @@ func (f *fundingManager) handleFundingCreated(fmsg *fundingCreatedMsg) {
 			deleteFromDatabase()
 		case <-f.quit:
 			// The fundingManager is shutting down, will resume
-			// wait for funding transaction on startup.
-		case <-doneChan:
+			// wait for funding transaction on startup.
+		case shortChanID, ok := <-confChan:
+			if !ok {
+				fndgLog.Errorf("waiting for funding confirmation failed")
```

Comment:
This line looks a bit long. We try to keep a 80 character line length width 8 width tabs.